### PR TITLE
SAT-21: Plotting function

### PR DESCRIPTION
**Pull request by hpatel0816**
**Changes:**
- Created a plotting function to display images using slider functionality

### CODE REVIEW

**Comment by hpatel0816** [comment]
Waiting on PR #11 to get merged to update my changes in the image_utils.py & test_image_utils.py files.

File: src/image_utils.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import os
-import cv2
 import numpy as np
+from PIL import Image
```
Comment (pijner) [inline src/image_utils.py:3]: Does Pillow need to be added to `requirements.txt`? I get a `ModuleNotFoundError: No module names 'PIL'` error when running with the current requirements

Comment (pijner) [reply src/image_utils.py:3]: Looks like matplotlib and ipywidgets need to be added to requirements as well

File: src/image_utils.py [hunk 2]
```diff
@@ -13,4 +15,50 @@ def read_tiff_from_file(file_path: str | os.PathLike) -> np.ndarray:
     Returns:
         np.ndarray: numpy array containing file contents. Assumes BGR format
     """
-    return cv2.imread(file_path)
+    pass # TODO: Finish this function
+
+
+def plot_tiff_images(dir: str | os.PathLike) -> None:
```
Comment (pijner) [inline src/image_utils.py:22]: This is awesome! Works really well once the requirements are updated

File: src/image_utils.py (hunk 1 above)
Comment (hpatel0816) [reply src/image_utils.py:3]: Thanks for pointing that out. I'll fix it up!

**Comment by hpatel0816** [comment]
@pijner How would you like me to test the unit tests for this function?

**Review by pijner — APPROVED** [review]
Don't think we need a unittest for this as it's a visualization helper. Good to merge now

**Pull request merged**